src/agent.js

```diff
@@ -30,9 +30,9 @@ const Articles = {
 
 const Comments = {
   populate: article_slug =>
-    requests.get("/articles/" + article_slug + "/comments")
-  // postComment: (article_slug, value) =>
-  //   requests.post("/articles/" + article_slug + "/comments/", value)
+    requests.get("/articles/" + article_slug + "/comments"),
```

Comment:
Template literal can make this more readable which typically prevent from forget `/` in the path:
```js
requests.get(`/articles/${article_slug}/comments`);
```